### ISSUE TITLE
Jump To Window - Overpopulated Collections List doesn't have scroll

### ISSUE THREAD

**Issue by takoz53** [ppy/osu-stable-issues#1274]
**Description:**
To be honest, I always wondered if this was forgotten and will be ever fixed.. I have encountered following problem multiple times: The "Jump To" Window is probably intented to give the user a faster way to jump to a specific song. BUT you can also select a playlist and by doing so, osu! will only play the songs inside your collection, making it a perfect music player, BUT if you overpopulate your collections, you can't access all of the collections anymore.
![screenshot2849](https://github.com/user-attachments/assets/4a3e4c75-e612-4e66-91ee-ff60cc626d49)
![screenshot2850](https://github.com/user-attachments/assets/d7db3f00-7a9a-4f20-8aca-0cf48f8f8a4a)
On comparison between SS1 and SS2, there is collections, which are not available in SS1, although I should be able to access them.

**Problem:**
* Can't access all collections inside the "Jump to" Window
* Only occurs when you have too many collections (when you hit ~25 collections at 1920x1080 if resolution matters here)
* Collections populate down the Y axis and go out of screen

**Solution:**
* Implement a scrollable list for the collections inside the "Jump to" window.

**Comment by peppy** [comment]
I don't think it will be fixed sorry. We'll make sure it doesn't happen on lazer though.

**Comment by takoz53** [comment]
Alright, thanks. Will be closing this issue then!

**Comment by peppy** [comment]
No need to close, just leave it to decay along with stable.
